.gitlab-ci.yml

```diff
@@ -103,174 +115,210 @@ build_wheel_macos_py38:
     - zsh -e scripts/test.sh --wheel-path=${WHEEL_PATH} --python=${PYTHON}
       --test-package=${TEST_PACKAGE} --requirements=${REQUIREMENTS} --fast
 
-test_macos11_py27_coremltools_test:
-  <<: *test_macos_pkg
-  tags:
-    - macos11
-  dependencies:
-    - build_wheel_macos_py27
-  variables:
-    WHEEL_PATH: build/dist/*cp27*10_16*
-    TEST_PACKAGE: coremltools.test
-    PYTHON: "2.7"
-  only:
-    changes:
-      - mlmodel/**/*
-      - coremltools/test/**/*.{py}
-      - coremltools/models/**/*.{py}
-      - coremltools/converters/caffe/**/*.{py}
-      - coremltools/converters/keras/**/*.{py}
-      - coremltools/converters/libsvm/**/*.{py}
-      - coremltools/converters/sklearn/**/*.{py}
-      - coremltools/converters/xgboost/**/*.{py}
 
-test_macos11_py37_coremltools_test:
+test_macos11_py37_coremltools_smoke_test:
   <<: *test_macos_pkg
   tags:
     - macos11
   dependencies:
     - build_wheel_macos_py37
   variables:
     WHEEL_PATH: build/dist/*cp37*10_16*
-    TEST_PACKAGE: coremltools.test
+    TEST_PACKAGE: coremltools.test.neural_network.test_keras2_numeric
```

Comment:
Looks like with this small set of tests, we can still get `RuntimeError: Error compiling model: "The file couldn’t be saved.".` (https://gitlab.com/zach_nation/coremltools/-/jobs/766272906). We should probably just write (or select if we have that) one network and one predict for the smoke test?

Reply:
Yeah that's what I thinking, to write a smoke test where we explicitly save the mlmodel, and then call predict.

Reply:
(btw, when we get this error `RuntimeError: Error compiling model: "The file couldn’t be saved.`, we are past the stage of loading the MLModel Proxy from the `.so`, so that is an indication that the wheels are correct and `libcoremlpython.so` can be loaded for prediction)

Reply:
> CI link: https://gitlab.com/zach_nation/coremltools/-/pipelines/196526785

@aseemw ConvTranspose test are failing and I also see timeout https://gitlab.com/zach_nation/coremltools/-/jobs/766002554